src/app/private/course/[courseId]/profile/feedback/page.tsx

```diff
@@ -0,0 +1,167 @@
+"use client";
+import Header from "@components/Header";
+import {
+  Button,
+  FormControl,
+  FormControlLabel,
+  IconButton,
+  Radio,
+  RadioGroup,
+  TextField,
+  Typography,
+} from "@mui/material";
+import { useRouter } from "next/navigation";
+import ArrowBackIcon from "@mui/icons-material/ArrowBack";
+import React from "react";
+import theme from "theme";
+import { addFeedback } from "@services/client/feedback";
+import { Timestamp } from "firebase/firestore";
+
+const Page = () => {
+  const router = useRouter();
+
+  const formControlLabelStyles = {
+    display: "flex",
+    flexDirection: "row-reverse",
+    alignItems: "center",
+    justifyContent: "space-between",
+    marginLeft: 0,
+    marginRight: 0,
+  };
+
+  const [recommendation, setRecommendation] = React.useState<number | null>(
```

Comment:
I would opt in for an enum for better semantics? We can also construct an object which maps the recommendation to label, so the actual `FormControlLabel` is cleaner.

```ts
enum Recommendation {
   VERY_UNLIKELY;
   ...
}

type RecommendationLabel = Record<Recommendation, String>

const recommendations: RecommendationLabel = { Recommendation.VERY_UNLIKELY: "Very unlikely" };
```

I guess technically, we can also use a constant string enum

```ts
enum Recommendation {
  VERY_UNLIKELY="Very unlikely"
}
```

but the tradeoff is space, so I'm good with either.

Reply:
Better yet,

```ts
type RecommendationLabel = { [key in Recommendation]: string };
```

enforces that all the enum value must be initializes